### ISSUE TITLE
text-based lookup tables

### ISSUE THREAD

**Issue by aminabedi68** [fontforge/fontforge#4925]
Hi
i think work with lookup tables(i mean line to line tables) should be much easier. it's very hard to input and edit data into them.
here is an idea: a text-based environment for write the afdko feature codes(like new fontlabs) or using XML data structure or even simple separators to compile text like table for fontforge. respectfully i think the biggest problem for this kind of issues is the priority, because fontforge developers didn't use it like regular users.

**Comment by skef** [comment]
It's already possible to import AFDKO feature files.

**Comment by aminabedi68** [comment]
yes, but I mean internal afdko editor WITH fontforge tiny gui tools like glyph description and picker... as i said above(for emphasis, elaborate and give you a different point of view): i think fontforge developers should use fontforge like a noob users too, i saw the power of python scripting or internal codes before but i spend a lot of time inside gui yet because i can't deal with tones of codepoint names... today i had to input a large amount of data through the gui, beacuse i needed the codepoint descriptions(showing characters in parents, one of the best fontforge's helpers) to see what is going on...It was really bad experience. this is not a personal problem(i have a lot of bigger problem with fontforge), no offence skef, i appreciate your works here, i really do, but I can't see a considerable improvement in user experience through this years. I think we need to fix bad experiences more than anything.

**Comment by skef** [comment]
As always, it's possible to code new functionality for alternate means of doing things but the challenge is finding people interested in doing the work.
